utils/channel_utils.jsx

```diff
@@ -2,17 +2,15 @@
 // See LICENSE.txt for license information.
 
 import {haveIChannelPermission} from 'mattermost-redux/selectors/entities/roles';
-import {getRedirectChannelNameForTeam as getRedirectChannelNameForTeamRedux} from 'mattermost-redux/selectors/entities/channels';
 import Permissions from 'mattermost-redux/constants/permissions';
 
-import store from 'stores/redux_store.jsx';
 import Constants from 'utils/constants';
 import * as Utils from 'utils/utils.jsx';
 
-export function canManageMembers(channel) {
+export function canManageMembers(channel, state) {
```

Comment:
Could you switch this around so that `state` is the first parameter? That way, it'll match all of the other selectors in case we ever want to memoize this using `createSelector`